Pinta.Core/Enumerations/MouseButton.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Pinta.Core
```

Comment:
Should this file have the standard license header? (same for the other new files)

Reply:
We might want to consider something simpler moving forward, but that's probably not a discussion for this PR.  Added the standard header to files.

(https://forums.dotnetfoundation.org/t/file-headers-and-copyright-statements/1276)